### DISCUSSION TITLE
mods的bugPCL咋不显示了

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#3192] [提问] [answered]
以前（v2.6.10)时，如果一个mod不兼容版本等问题，在你退出fabric API时PCL会弹出一个弹窗`MC出现bug，解决方法……，看不懂用翻译`现在只有一个弹窗，没有解决办法了，是咋回事

**Comment by allMagicNB** [comment]
显示不准确（

**Comment by WForst-Breeze** [comment]
如果单纯是模组版本问题感觉应该是能分析出来错误问题的，准备日志和崩溃报告交个综合 bug 反馈?

Edit: 试了一下确实不行，我也记得以前能分析出来这类错误，但使用了多个 PCL 过往版本测试后发现都分析不出来（）记忆错乱了草

**Comment by allMagicNB** [comment]
> 如果单纯是模组版本问题感觉应该是能分析出来错误问题的，准备日志和崩溃报告交个综合 bug 反馈

“看不懂用翻译”，我想这前面会不会是Mod本身输出的内容？

**Reply by WForst-Breeze** [reply]
版本不兼容的话 Fabric 是有弹窗报错提示的（错误报告中也会包含弹窗中的提示），正常讲 PCL 也会将 Fabric 弹窗中的版本不兼容报错内容直接列在 PCL 的崩溃弹窗中。看不懂用翻译的意思是这句话是以原始文本（英文）直接被搬到 PCL 报错窗口的，需要用翻译器才能知道这句报错的意思

**Comment by ghost** [comment]
不是的，应该是PCL输出的，我都是Fabric，玩Forge的有提示吗？

**Reply by WForst-Breeze** [reply]
没太看懂想表达啥意思，什么是 “PCL 输出的”🤔

**Reply by ghost** [reply]
PCL自己的建议

**Comment by ghost** [comment]
而且PCL要翻译的话它也不会只翻这一句吧，只要Forge也没有提示，就交一个Issues

**Comment by allMagicNB** [comment]
> 而且PCL要翻译的话它也不会只翻这一句吧，只要Forge也没有提示，就交一个Issues

应该做了分析，但没做完（？

**Comment by WForst-Breeze** [comment, marked as answer]
后续讨论请移步 #3228
